src/game_of_life/main.cpp

```diff
@@ -18,28 +18,27 @@
  *
  **************************************************************************/
 
-#include "sim.hpp"
-
-#include <Magnum/Platform/Sdl2Application.h>
 #include <Magnum/GL/DefaultFramebuffer.h>
 #include <Magnum/GL/Mesh.h>
 #include <Magnum/GL/Texture.h>
 #include <Magnum/GL/TextureFormat.h>
-#include <Magnum/PixelFormat.h>
 #include <Magnum/ImageView.h>
-#include <Magnum/Shaders/FlatGL.h>
-#include <Magnum/Primitives/Square.h>
 #include <Magnum/MeshTools/Compile.h>
+#include <Magnum/PixelFormat.h>
+#include <Magnum/Platform/Sdl2Application.h>
+#include <Magnum/Primitives/Square.h>
+#include <Magnum/Shaders/FlatGL.h>
 #include <Magnum/Trade/MeshData.h>
 
+#include <chrono>
 #include <sycl/sycl.hpp>
```

Comment:
IMO this should be moved to its own block like this suggests: https://blog.knatten.org/2010/07/01/the-order-of-include-directives-matter/

Reply:
Same as in  `fluid.h` - apparently `clang-format` treats sycl.hpp as an STL include and puts it in the same block with other STL includes. Looks like a `clang-format` bug to me.

Also, I fully agree with the ordering in that blog post and always use that myself, but the Google style guide applied here [instructs otherwise](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes). This is why the sim.hpp include got moved from the first to the last position.

Reply:
Resolved by adding our own `.clang-format` style file based on the Google style. The new style overrides the include categories with a separate category for SYCL includes.